recipe/meta.yaml

```
@@ -125,33 +54,162 @@ build:
 
 # Required for conda to generate the correct build matrix
 requirements:
```

You could anchor this, like so:

```suggestion
 requirements: &requirements
```